test/util/edu-helpers.js

```diff
@@ -16,6 +18,35 @@ function completeVeteranInformation(client, data, onlyRequiredFields) {
   }
 }
 
+function completeRelativeInformation(client, data, onlyRequiredFields) {
+  const dobFields = data.relativeDateOfBirth.split('-');
+  client
+    .clearValue('input[name="root_relativeFullName_first"]')
+    .setValue('input[name="root_relativeFullName_first"]', data.relativeFullName.first)
+    .clearValue('input[name="root_relativeFullName_last"]')
+    .setValue('input[name="root_relativeFullName_last"]', data.relativeFullName.last)
+    .clearValue('input[name="root_relativeSocialSecurityNumber"]')
+    .setValue('input[name="root_relativeSocialSecurityNumber"]', data.relativeSocialSecurityNumber)
+    .clearValue('input[name="root_relativeDateOfBirthYear"]')
+    .setValue('input[name="root_relativeDateOfBirthYear"]', parseInt(dobFields[0], 10).toString())
+    .click('input[name="root_relationship_1"]');
+  selectDropdown(client, 'root_relativeDateOfBirthMonth', parseInt(dobFields[1], 10).toString());
+  selectDropdown(client, 'root_relativeDateOfBirthDay', parseInt(dobFields[2], 10).toString());
+
+  if (!onlyRequiredFields) {
+    client
+      .setValue('input[name="root_relativeFullName_middle"]', data.relativeFullName.middle)
+      .click(data.gender === 'M' ? 'input[name=root_gender_0' : 'input[name=root_gender_1');
+    selectDropdown(client, 'root_relativeFullName_suffix', data.relativeFullName.suffix);
+  }
+}
+
+function completeAdditionalBenefits(client, data) {
+  client
+    .click(data.nonVaAssistance ? 'input[name="root_nonVaAssistanceYes"]' : 'input[name="root_nonVaAssistanceNo"]')
+    .click(data.civilianBenefitsAssistance ? 'input[name="root_civilianBenefitsAssistanceNo"]' : 'input[name="root_civilianBenefitsAssistanceNo"]');
```

Comment:
Oops, thanks! Will push that change once I see how jenkins behaves on this last commit.